app/view/LayerTree.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * A Layertree which autoconnects to the map of the application.
+ */
+Ext.define('CpsiMapview.view.LayerTree', {
+    extend: 'Ext.tree.Panel',
+    xtype: 'layertree',
```

Comment:
Please use the prefix `cmv_` for the xtype.

Reply:
Will do.